vitest.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    coverage: {
+      provider: "istanbul",
+      reporter: ["text", "json", "html"],
```

Comment:
what does this mean? If its the kind of file formatas that are supported: why do we not need to specify js / ts?

Reply:
```
npx vitest run --coverage --reporter=json
```

to see the json formatted output. but seems to work without this.